src/js/components/ServicesTable.js

```diff
@@ -118,23 +115,25 @@ var ServicesTable = React.createClass({
       );
     }
 
-    if (service instanceof Framework) {
-      // Get framework image/icon
-      itemImage = (
-        <span
-          className="icon icon-small icon-image-container icon-app-container icon-margin-right">
-          <img src={service.getImages()['icon-small']}/>
-        </span>
-      );
-    }
+    // Get service image/icon
+    return (
+      <span
+        className="icon icon-small icon-image-container icon-app-container icon-margin-right">
+        <img src={service.getImages()['icon-small']}/>
+      </span>
+    );
+  },
```

Comment:
i think what PG is saying is that if we can implement a `getImages()` onto the `ServiceTree` so that we don't even have to do the check and it'll just work for all instances.

but PG i don't think that's possible at the moment because `ServiceTree` uses the `Icon` component and everything else uses the `<img src=` element